.github/workflows/ci_tests.yml

```diff
@@ -0,0 +1,126 @@
+name: CI Tests
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  pull_request:
+  schedule:
+    # run every Monday at 4am UTC
+    - cron: '0 4 * * 1'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  TOXARGS: '-v'
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    name: ${{ matrix.prefix }} ${{ matrix.os }}, ${{ matrix.tox_env }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.allow_failure }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            python: '3.9'
+            tox_env: 'py39-test'
+            allow_failure: false
+            prefix: ''
+
+          - os: ubuntu-latest
+            python: '3.10'
+            tox_env: 'py310-test'
+            allow_failure: false
+            prefix: ''
+
+          - os: ubuntu-latest
+            python: '3.11'
+            tox_env: 'py311-test-cov'
+            toxposargs: --remote-data=any
+            allow_failure: false
+            prefix: '(Temp allowed failure)'
```

Comment:
These two lines are not consistent with one another.

Reply:
Fixed